src/jit/emitarm64.cpp

```diff
@@ -559,6 +559,16 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(isVectorRegister(id->idReg1()));
             break;
 
+        case IF_DV_1D: // DV_1D   .Q.........mmmmm .ll...nnnnnddddd      Vd,{ V... },Vm   (tbl - structure load store)
```

Comment:
The naming convention used for instruction formats require this to be named starting with DV_3 as it encodes three registers